contracts/staking/StakeManager.sol

```diff
@@ -344,6 +421,7 @@ contract StakeManager is Validator, IStakeManager, RootChainable, Lockable {
     DYNASTY = newDynasty;
```

Comment:
Don't use the constant typing `DYNASTY ` since it's not a constant.

Reply:
done